lib/monitor.js

```diff
@@ -103,7 +118,7 @@ Monitor.prototype.stopSocketsMonitor = function () {
  * module.
  * @class
  */
-function Timer(callback, start) {
+function Timer (callback, start) {
```

Comment:
this is definitely one of those things that everyone has a personal preference on, but for the mostpart in the API we have that space between the name and args (or between `function` and the args) when defining functions. now, for every one that you show me that doesn't have the space I can show you two that don't, so it's really just up to preference and consistency at this point... maybe some day we'll enforce it a little more...